plugins/hu.elte.txtuml.utils/src/hu/elte/txtuml/utils/NotifierOfTermination.java

```diff
@@ -8,10 +8,6 @@
 	private final List<Runnable> list = new ArrayList<>();
 	private boolean terminated = false;
 
-	public synchronized boolean isTerminated() {
-		return terminated;
-	}

```

Comment:
It behaves like a notifier, an event source, so it should only communicate that way.